datumaro/util/test_utils.py

```diff
@@ -263,3 +265,37 @@ def compare_dirs(test, expected: str, actual: str):
 def run_datum(test, *args, expected_code=0):
     from datumaro.cli.__main__ import main
     test.assertEqual(expected_code, main(args), str(args))
+
+@contextlib.contextmanager
+def mock_tfds_data(example=None):
```

Comment:
Probably, we need to place such functionality (mocks, fixtures, pytest-dependents etc.) in the `tests/` directory. `run_datum` is also a  good candidate for moving. This is not a blocker, but we should do it at some point.